File: src/main/java/de/cloudypanda/main/adventcalendar/config/AdventCalendarDayConfig.java
package de.cloudypanda.main.adventcalendar.config;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@Getter
@Setter
public class AdventCalendarDayConfig {
    private String date;
    private String message;
    private int points;
    private AdventCalendarSubmitItemConfig itemToSubmit;
}

File: src/main/java/de/cloudypanda/main/adventcalendar/config/AdventCalendarDayConfig.kt
package de.cloudypanda.main.adventcalendar.config;

data class AdventCalendarDayConfig(
    val date: String,
    val message: String,
    val points: Int,
    val challengeType: AdventCalendarChallengeType,
    val achievementToSubmit: AdventCalendarSubmitAchievementConfig?,
    val itemToSubmit: AdventCalendarSubmitItemConfig?
) {
    constructor() : this("", "", 0, AdventCalendarChallengeType.ITEM, null, null)
}
